event_webhook_types_test.go

```diff
@@ -60,10 +60,87 @@ func TestBuildEventUnmarshal(t *testing.T) {
 	}
 
 	if event.BuildCreatedAt != "2021-02-23T02:41:37.886Z" {
-		t.Errorf("BuildCreatedAt is %s, want %s", event.User.Name, expectedName)
+		t.Errorf("BuildCreatedAt is %s, want 2021-02-23T02:41:37.886Z", event.BuildCreatedAt)
 	}
 }
 
+func TestJobEventUnmarshal(t *testing.T) {
+	jsonObject := loadFixture("testdata/webhooks/job.json")
+
+	var event *JobEvent
+	err := json.Unmarshal(jsonObject, &event)
+	if err != nil {
+		t.Errorf("Job Event can not unmarshaled: %v\n ", err.Error())
+	}
+
+	if event == nil {
+		t.Errorf("Job Event is null")
+	}
+
+	expectedEvent := JobEvent{
+		ObjectKind:         "build",
+		Ref:                "main",
+		Tag:                false,
+		BeforeSHA:          "0000000000000000000000000000000000000000",
+		SHA:                "95d49d1efbd941908580e79d65e4b5ecaf4a8305",
+		BuildID:            3580121225,
+		BuildName:          "auto_deploy:start",
+		BuildStage:         "coordinated:tag",
+		BuildStatus:        "success",
+		BuildCreatedAt:     "2023-01-10 13:50:02 UTC",
+		BuildStartedAt:     "2023-01-10 13:50:05 UTC",
+		BuildFinishedAt:    "2023-01-10 13:50:54 UTC",
+		BuildDuration:      49.503592,
+		BuildAllowFailure:  false,
+		BuildFailureReason: "unknown_failure",
+		RetriesCount:       1,
+		PipelineID:         743121198,
+		ProjectID:          31537070,
+		ProjectName:        "John Smith / release-tools-fake",
+		User: &EventUser{
+			ID:        2967854,
+			Name:      "John Smith",
+			Username:  "jsmithy2",
+			AvatarURL: "https://gitlab.com/uploads/-/system/user/avatar/2967852/avatar.png",
+			Email:     "john@smith.com",
+		},
+		Repository: &Repository{
+			Name:              "release-tools-fake",
+			Description:       "",
+			WebURL:            "",
+			AvatarURL:         "",
+			GitSSHURL:         "git@gitlab.com:jsmithy2/release-tools-fake.git",
+			GitHTTPURL:        "https://gitlab.com/jsmithy2/release-tools-fake.git",
+			Namespace:         "",
+			Visibility:        "",
```

Comment:
There is an existing bug that seems to prevent the `Repository.Visibility` attribute from being unmarshaled correctly.